360/watchapp/templates/watchapp/homepage.html

```diff
@@ -41,19 +74,125 @@
         .header-buttons button {
             background-color: transparent;
             border: none;
-            color: black;
+            color: white;
             cursor: pointer;
+            font-family: inherit;
             font-size: 18px;
             display: flex;
             align-items: center;
         }
-        .header-buttons button:hover {
```

Comment:
Do we not need this anymore?

Reply:
the hover functionality of the buttons is still here, it's just in a different place in the code. its now on line 102-103 with some other stylistic features